front/app/components/EsriMap/components/ResetMapViewButton.tsx

```diff
@@ -54,35 +55,30 @@ const ResetMapViewButton = ({ mapConfig, mapView }: Props) => {
   };
 
   return (
-    <Box zIndex="999999">
-      <Box
-        zIndex="1000"
-        mt="-72px"
-        ml="16px"
-        w="36px"
-        background={colors.white}
+    <Box ref={resetButtonRef}>
+      <Tooltip
```

Comment:
By adding a ref, we're able to grab this component in the parent component after loading and put it into the Esri Map itself as a UI element.